packages/contractkit/src/wrappers/Exchange.ts

```diff
@@ -123,4 +123,19 @@ export class ExchangeWrapper extends BaseWrapper<Exchange> {
    * @return The corresponding cUsd amount.
    */
   quoteGoldBuy = (buyAmount: NumberLike) => this.getSellTokenAmount(buyAmount, true)
+
+  /**
+   * Returns the exchange rate estimated at buyAmount.
+   * @param sellGold `true` if gold is the sell token
+   * @param buyAmount The amount of buyToken to estimate the exchange rate at (optional,
+   * defaults to large buyAmount to show worst case rate)
+   * @return The exchange rate (number of sellTokens received for one buyToken).
+   */
+  async getExchangeRate(
+    sellGold: boolean,
```

Comment:
can we add getUsdExchangeRate and getGoldExchangeRate.
I think it provides a better devux than the boolean, we can leave it too

Reply:
We've done the same with `quoteGoldBuy`, `quote...` variants

Reply:
Good idea will follow the pattern and add those functions